tex/alg-NT/arithmetic.tex

```diff
@@ -0,0 +1,357 @@
+\chapter{Dirichlet's theorem on arithmetic progressions}
+
+\section{Introduction}
+
+In this chapter, we will sketch a proof of the following theorem:
+\begin{theorem}[Dirichlet's theorem on arithmetic progressions]
+	For every positive integers $a, n > 0$ such that $\gcd(a, n) = 1$, then there are infinitely
```

Comment:
should this say something like every pair of positive integers?

Reply:
just a wording issue. I don't mind either, but there's another incident where the other wording is used

![image](https://github.com/user-attachments/assets/0e486361-2eef-4577-895b-f02ba7bb016a)

Reply:
i would say 16F would be better as "for all $x_1, x_2 \in X" as well, so I'll change that now.

Reply:
just updated in 80b056a81ec211d2b5b9626105d0634e6acc8c30